db/deploy/tables/attachment.sql

```diff
@@ -29,10 +29,23 @@ perform ccbc_private.grant_permissions('select', 'attachment', 'ccbc_auth_user')
 perform ccbc_private.grant_permissions('insert', 'attachment', 'ccbc_auth_user');
 perform ccbc_private.grant_permissions('update', 'attachment', 'ccbc_auth_user');
 
+end
+$grant$;
+
+-- Enable row-level security
+alter table ccbc_public.attachment force row level security;
+alter table ccbc_public.attachment enable row level security;
 
+do
+$policy$
+begin
+-- ccbc_auth_user RLS: can see and modify only its own records
+perform ccbc_private.upsert_policy('ccbc_auth_user_select_attachment', 'attachment', 'select', 'ccbc_auth_user', 'application_id in (select id from ccbc_public.applications where owner=(select sub from ccbc_public.session()))');
+perform ccbc_private.upsert_policy('ccbc_auth_user_insert_attachment', 'attachment', 'insert', 'ccbc_auth_user', 'application_id in (select id from ccbc_public.applications where owner=(select sub from ccbc_public.session()))');
+perform ccbc_private.upsert_policy('ccbc_auth_user_update_attachment', 'attachment', 'update', 'ccbc_auth_user', 'application_id in (select id from ccbc_public.applications where owner=(select sub from ccbc_public.session()))');
```

Comment:
Since owner is being deprecated should we use created_by?

Reply:
`main` still has `owner` - https://github.com/bcgov/CONN-CCBC-portal/blob/main/db/deploy/tables/applications.sql